packages/rich-text/src/plugins/Hyperlink/components/EntityHyperlink.tsx

```diff
@@ -1,15 +1,17 @@
 import * as React from 'react';
 
 import { FieldAppSDK, Link } from '@contentful/app-sdk';
-import { Tooltip, TextLink } from '@contentful/f36-components';
+import { Text } from '@contentful/f36-components';
+import { Path } from 'slate';
```

Comment:
should we import this from src/internal/queries?
because we wrap there `plate` and `slate`

Reply:
(`isChildPath`)